open_dread_rando/files/schema.json

```diff
@@ -313,9 +313,36 @@
                                     "type": "boolean",
                                     "default": false,
                                     "description": "Enables an in-game death counter shown in the player's HUD"
+                                },
+                                "enable_room_name_display": {
+                                    "type": "string",
+                                    "enum": [
+                                        "Never",
+                                        "Always",
+                                        "When entering a new room"
```

Comment:
While this value does work, it has a very "user friendly" kind instead of a more "internal-value".

I'm guessing this is because you ended up using the enum value as the GUI element over at rdv. Having that leak into the API in odr is a sign something's wrong :)

Reply:
see the comment I made on that PR, it shares how we usually do the descriptive names there for GUI.

Other than that, the general rule is that we design the patchers first and then adapt randovania to the patcher, not the opposite!